python_tests/artifact/test_backend.py

```diff
@@ -89,19 +89,18 @@ def test_list_trial_artifacts(init_storage_with_artifact_meta: MagicMock) -> Non
     ]
 
 
-def test_artifact_store_none() -> None:
+def test_study_artifact_store_none() -> None:
     storage = optuna.storages.InMemoryStorage()
     app = create_app(storage)
     status, _, body = send_request(
         app,
         "/artifacts/0/0",
         "GET",
-        content_type="application/json",
```

Comment:
I think that this is unnecessary because the content is not JSON.

Reply:
That's true 👍 Good catch.